platforms/hyperledger-besu/configuration/roles/create/certificates/ambassador/tasks/nested_main.yaml

```diff
@@ -16,6 +16,39 @@
     path: "{{ ambassadortls }}"
     check: "ensure_dir"
 
+# Create ambassador tls certificate by cert-manager
+- name: create Ambassador default tls certificate by cert-manager for "{{ node_name }}"
+  k8s:
+    state: present
+    kubeconfig: "{{ kubernetes.config_file }}"
+    definition:
+      apiVersion: cert-manager.io/v1
+      kind: Certificate
+      metadata:
+        annotations:
+          cert-manager.io/issue-temporary-certificate: "true"
+          acme.cert-manager.io/http01-edit-in-place: "true"
+        name: "{{ node_name }}-ambassador-certs"
+        namespace: "{{ component_ns }}"
+      spec:
+        secretName: "{{ node_name }}-ambassador-certs"
+        issuerRef:
+          name: letsencrypt-prod-{{ component_ns }}
+          kind: Issuer
+        duration: 8760h #365 days
+        renewBefore: 168h #7 days
+        isCA: false
+        dnsNames:  
+          - "{{ node_name }}.{{ organizationItem.external_url_suffix }}"
```

Comment:
should be like this {{ node_name }}.{{ component_ns }}.{{ organizationItem.external_url_suffix }}, since node urls have the namespace in them.

Reply:
Will include the suggested dns name as well